modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/TabItem.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {LAYOUT_DATA_ITEM_TYPES, updateUsedWidget} from './AddPanel';
+import {
+	usePlidContext,
+	useSetWidgetsContext,
+	useWidgetsContext,
+} from './AddPanelContext';
+import {addPortlet, useDragSymbol} from './useDragAndDrop';
+
+import 'product-navigation-control-menu/css/TabItem.scss';
+
+const addItem = ({item, plid, setWidgets, widgets}) => {
+	const targetItem = document.querySelector('.portlet-dropzone');
+
+	if (!item.used) {
```

Comment:
Maybe we don't need to check `item.used` here, as the button is only available if the item is not disabled.

Reply:
You're right, I have removed this.